src/tv2_afvd_showstyle/helpers/pieces/evaluateCues.ts

```diff
@@ -51,7 +51,8 @@ export async function EvaluateCues(
 			EvaluateCueGraphic,
 			EvaluateCueBackgroundLoop,
 			EvaluateCueGraphicDesign: EvaluateCueDesign,
-			EvaluateCueGraphicSchema,
+			EvaluateCueGraphicSchema: (c, p, partId, parsedCue) =>
```

Comment:
From this I have no idea what `c` and `p` are. Would be nice with describing names.

Reply:
`c` and `p` have been renamed to `cue` and `piece`.